doc/whats_new/v0.22.rst

```diff
@@ -347,6 +348,12 @@ Changelog
   requires less memory.
   :pr:`14108`, pr:`14170`, pr:`14296` by :user:`Alex Henrie <alexhenrie>`.
 
+- |Efficiency| :class:`linear_model.RidgeCV` now does not allocate a potentially
+  large array to store dual coefficients for all hyperparameters during its
+  `fit`, nor an array to store all LOO predictions unless `store_cv_values` is
+  `True`.
```

Comment:
I see no test about both these claims?

Reply:
This is difficult to test since they were variable allocated locally in the `fit` function. The current implementation will only store the values at each iteration if necessary.